Swona/Sources/Swona/Parser.swift

```diff
@@ -357,6 +359,21 @@ public class Parser {
         return Expression.`if`(condition: condition, consequent: consequent, alternative: alternative, location: location)
     }
 
+    private func parseUnless() throws -> Expression {
+        let location = try lexer.expect(expected: .keyword(.unless))
+        let condition = try inParens { try parseTopLevelExpression() }
+        let consequent = try parseTopLevelExpression()
+        let alternative: Expression?
+        if try lexer.readNextIf(token: .keyword(.else)) {
+            alternative = try parseTopLevelExpression()
+        }
+        else {
+            alternative = nil
+        }
+
+        return Expression.`if`(condition: .not(exp: condition, location: location), consequent: consequent, alternative: alternative, location: location)
```

Comment:
Key difference between if and unless is here where it uses if but includes a .not around the first condition.